### ISSUE TITLE
Intro - Regression Analysis

### ISSUE THREAD

**Issue by kunwar89** [team-one-eee-511/proj3-time-series-forecasting#1]
1. Cleaned up Earlier Linear Regression Analysis 
2. Using Keras NN for same Dimension features
3. Result : Mean Absolute Error better by about 2
4. Validation Loss per Epochs not implemented
5. Current Activation is ReLU
6. **External Test Dataset Analysis Not Implemented**
